include/eve/detail/skeleton.hpp

```diff
@@ -79,12 +83,12 @@ namespace eve::detail
     template<typename Func, typename Idx, typename... Ts>
     EVE_FORCEINLINE auto operator()(Func &&fn, Idx const &i, Ts &&... vs) const noexcept
     {
-      return EVE_FWD(fn)(at(EVE_FWD(vs), i)...);
```

Comment:
can we name at as smth longer? like `get_at`? I don't like adl look upabale name that is this short and not part of an api

Reply:
I actually changed it to be fully qualified. But I can rename no pb

Reply:
Renamed to get_at